tests/nfa/nfa.cc

```diff
@@ -1188,7 +1189,7 @@ TEST_CASE("Mata::Nfa::complement()")
 
 		cmpl = complement(aut, alph, {{"algorithm", "classical"},
                                     {"minimize", "false"}});
-        Nfa empty_string_nfa{ Mata::Nfa::create_sigma_star_nfa(&alph) };
+        Nfa empty_string_nfa{ Mata::Nfa::Builder::create_sigma_star_nfa(&alph) };
```

Comment:
Seeing how this is used I'm thinking that maybe we could change it to `Mata::Nfa::Build` and function `sigma_star_nfa`, etc.

Then the call will be `Mata::Nfa::Build::sigma_star_nfa(&alhp)`, which for me i smore elegant. Consider.

Reply:
I thought about it, too. However, we have other functions such as the `construct()` functions. Later, we might add a function to construct an NFA from Mata format directly. That would probably also belong to the builder. Then you would have something as `Mata::Nfa::Build::construct()`, even if `Mata::Nfa::Build::nfa_from_mata_format()` would work.

Furthermore, namespace names should be nouns, not verbs. I would keep `Builder`. I find it even superior to `Factory`. It states more clearly what is the namespace for, in my opinion. `Factory` feels a bit ambiguous to me.

But we could think about shortening the names of `create_` functions. They are now in the `Builder` namespace. Therefore, we do not need the prefix `create_`, theoretically, because the expected call would be `Nfa::Builder::sigma_star_nfa()`. It reads quite nicely. But it depends. One might want to keep the `create_` prefix so that when the function is called without the namespace, you know what it does… Hard to say.